src/server/system_services/bucket_server.js

```diff
@@ -1142,20 +1142,32 @@ function get_cloud_buckets(req) {
                 return storage.getBuckets()
                     .then(data => data[0].map(bucket =>
                         _inject_usage_to_cloud_bucket(bucket.name, connection.endpoint, used_cloud_buckets)));
-            } else { //else if AWS
+            } else if (connection.endpoint_type === 'AWSSTS' || connection.endpoint_type === 'AWS') {
```

Comment:
@jackyalbo Do you think adding an else here should work instead of checking both endpoint types since it is failing for endpoint type `S3_COMPATIBLE` in the unit tests?

Reply:
yes, good catch by the unit-tests - we missed compatible